Project/pomodoro.md

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
add to `projects/python/pomodoro.md` or whatever language you're planning to use